fe/fe-core/src/main/java/com/starrocks/common/FeMetaVersion.java

```diff
@@ -199,6 +199,8 @@ public final class FeMetaVersion {
     public static final int VERSION_91 = 91;
     // for mysql external table support resource
     public static final int VERSION_92 = 92;
+    // for broker load support update_condition
+    public static final int VERSION_93 = 93;
     // note: when increment meta version, should assign the latest version to VERSION_CURRENT
-    public static final int VERSION_CURRENT = VERSION_92;
+    public static final int VERSION_CURRENT = VERSION_93;
```

Comment:
This may have an incompatibility error. I think you did not need the version.
I think you can write in the JSON format to reduce it.